frontend/src/component/feature/FeatureView/FeatureOverview/FeatureOverviewSidePanel/FeatureOverviewSidePanelDetails/FeatureOverviewSidePanelDetails.test.tsx

```diff
@@ -122,6 +163,46 @@ test('delete dependency', async () => {
 
     const deleteButton = await screen.findByText('Delete');
     userEvent.click(deleteButton);
+
+    await screen.findByText('Dependency removed');
+});
+
+test('delete dependency with change request', async () => {
+    setupChangeRequestApi();
+    render(
+        <>
+            <ToastRenderer />
+            <FeatureOverviewSidePanelDetails
+                feature={
+                    {
+                        name: 'feature',
+                        project: 'default',
+                        dependencies: [{ feature: 'some_parent' }],
+                        children: [] as string[],
+                    } as IFeatureToggle
+                }
+                header={''}
+            />
+        </>,
+        {
+            permissions: [
+                /* deliberately no permissions */
```

Comment:
this is how we can test change request permissions

Reply:
I deliberately omit the permissions and we should still be able to add things to change request